mmv1/products/apigee/terraform.yaml

```diff
@@ -47,11 +47,47 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       min_version: beta
       # Resource creation race
       skip_vcr: true
-
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/apigee_organization.go.erb
       encoder: templates/terraform/encoders/apigee_organization.go.erb
-
+  Instance: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["{{org_id}}/instances/{{name}}", "{{org_id}}/{{name}}"]
+    examples:
+    - !ruby/object:Provider::Terraform::Examples
+      name: "apigee_instance_basic"
+      skip_test: true
+    - !ruby/object:Provider::Terraform::Examples
+      # This is a more verbose version of the above that creates all
+      # the resources needed for the acceptance test.
+      name: "apigee_instance_basic_test"
```

Comment:
Having separate `*_basic` and `*_basic_test` examples seems a bit odd but is in line with what the other apigee resources do. Do you happen to know the reasoning behind this split?

Reply:
The reason is that for the tests only, we need to create the org in its own project (since there can only be one org per project), although this is irrelevant in terms of documentation. We don't want to confuse the user that they need a new project to create the instance, but we also don't want to create the org on the acceptance test project as that may cause multiple simultaneous acceptance tests run to compete creating the (only) org for that project.